src/main/java/com/coherentsolutions/training/automation/java/web/urnezaite/CartPage.java

```diff
@@ -0,0 +1,37 @@
+package com.coherentsolutions.training.automation.java.web.urnezaite;
+
+import com.coherentsolutions.training.automation.java.web.urnezaite.util.Log;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class CartPage extends BasePage {
+
+    @FindBy(css = "a.home")
+    private WebElement homeButton;
+
+    @FindBy(css = "td.cart_description small.cart_ref")
+    List<WebElement> cartItems;
+
+
+    private List<String> namesOfItemsInCart = new ArrayList<>();
```

Comment:
why do you create the field of the class if you use it only in on method? You can create this variable in the method and there